OSDOCS-8104: Adding OVS drops type to Packet drops (#66047)
openshift/openshift-docs: modules/network-observability-pktdrop-overview.adoc

* Top X dropped cause
* Top X flow dropped rates stacked with total

> Two kinds of packet drops are detected by Network Observability: host drops and OVS drops. Dropped flows are shown in the side panel of the *Traffic flows* table. Host drops are prefixed with `SKB_DROP` and OVS drops are prefixed with `OVS_DROP`, and a link to a description of the drop type is also available in the side panel. Examples of host drop reasons are as follows:

Reviewer: Suggestion to pair up drop types and their prefixes before describing the side panel contents:
```suggestion
Two kinds of packet drops are detected by Network Observability: host drops and OVS drops. Host drops are prefixed with `SKB_DROP` and OVS drops are prefixed with `OVS_DROP`. Dropped flows are shown in the side panel of the *Traffic flows* table along with a link to a description of each drop type. Examples of host drop reasons are as follows:
```

Author: Good suggestion! I'll take it :)

> Two kinds of packet drops are detected by Network Observability: host drops and OVS drops. If any flows are dropped, they are shown in the side panel of the Traffic flow table. Host drops are prefixed with `SKB_DROP` and OVS drops are prefixed with `OVS_DROP`, and a link to a description of the drop type is also available in the side panel. Examples of host drop reasons are as follows:

Reviewer 2: The same comment as above, no need of the second line

Author: I can see how the previous one wasn't needed because I basically said right before that more info about packet drops is shown in the side panel. But here, that previous mention of the side panel isn't there. DO you see what I mean? Maybe its still an obvious thing that needs to be removed 🤷‍♀️

> * `SKB_DROP_REASON_NO_SOCKET`: the packet dropped because the socket was not found. 

Reviewer 2: would `packet drop due to missing socket` sound better?

Author: Yeah I can get down with that.

> * `SKB_DROP_REASON_TCP_CSUM`: there is a TCP checksum error.

Reviewer 2: and `packet dropped due to TCP checksum error` sound better here?

Examples of OVS drops reasons are as follows:

> * `OVS_DROP_LAST_ACTION`: OVS dropped because a flow was configured with an implicit drop action, for example configuring network-policy

Reviewer 2: How does `OVS packets dropped due to implicit drop action, for example due to a network policy configured` sound?

> * `OVS_DROP_IP_TTL`: OVS dropped because IP TTL expired

Reviewer 2: `OVS packets dropped due to expired IP TTL`
